css/styles.css

```diff
@@ -42,38 +39,22 @@ h1 {
 
 .menu {
   width: 100%;
-  display: flex;
-  justify-content: space-around;
-  /* flex-direction: row; zu welchem elemnt?*/
   flex-wrap: wrap;
   position: fixed;
   border: solid;
   border-width: 2px;
   width: 100%;
-  display: flex;
-  justify-content: space-around;
-  /* flex-direction: row; zu welchem element?*/
-  flex-wrap: wrap;
-  position: fixed;
-  border: solid;
-  border-width: 2px;
   border-color: rgb(68, 115, 115);
   border-radius: 4px;
   background-color: rgb(68, 115, 115);
   font-size: 20px;
 }
 
```

Comment:
very very good solution! I like the look that the menu is fixed on top of the heading and scrolls down !